client/my-sites/plans/ecommerce-trial/business-upgrade-confirmation/index.tsx

```diff
@@ -1,60 +1,73 @@
+import { StepContainer } from '@automattic/onboarding';
 import { useTranslate } from 'i18n-calypso';
+import page from 'page';
+import { useLayoutEffect } from 'react';
 import QueryJetpackPlugins from 'calypso/components/data/query-jetpack-plugins';
-import QuerySitePlans from 'calypso/components/data/query-site-plans';
 import Main from 'calypso/components/main';
 import BodySectionCssClass from 'calypso/layout/body-section-css-class';
 import PageViewTracker from 'calypso/lib/analytics/page-view-tracker';
 import BusinessTrialIncluded from 'calypso/my-sites/plans/current-plan/trials/business-trial-included';
-import { useSelector } from 'calypso/state';
-import { isRequestingSitePlans } from 'calypso/state/sites/plans/selectors';
+import { useDispatch, useSelector } from 'calypso/state';
+import { hideMasterbar, showMasterbar } from 'calypso/state/ui/actions';
 import { getSelectedSite } from 'calypso/state/ui/selectors';
-import type { AppState } from 'calypso/types';
 
 import './style.scss';
 
+const noop = () => {};
+
 const BusinessUpgradeConfirmation = () => {
 	const selectedSite = useSelector( getSelectedSite );
+	const dispatch = useDispatch();
 	const translate = useTranslate();
 
-	const isFetchingSitePlan = useSelector( ( state: AppState ) => {
-		if ( ! selectedSite?.ID ) {
-			return false;
-		}
-		return isRequestingSitePlans( state, selectedSite.ID );
-	} );
+	useLayoutEffect( () => {
+		dispatch( hideMasterbar() );
 
-	const currentPlanName = isFetchingSitePlan ? '' : selectedSite?.plan?.product_name_short ?? '';
+		return () => {
+			dispatch( showMasterbar() );
+		};
+	}, [ dispatch ] );
```

Comment:
The masterbar hides the "Skip to dashboard" button, that's why I'm removing it.

I don't think there is a problem doing that since the checkout screen doesn't have the masterbar as well.

Reply:
I wish we had a nicer declarative way to hide the top bar. That is aside from adding our custom route to the big list of excluded routes that is 😮‍💨